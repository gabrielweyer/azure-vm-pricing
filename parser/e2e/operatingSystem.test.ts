import assert from "./assertExtensions";

describe('End-to-end tests for supported operating systems', () => {
  const culture = 'en-us';
  const currency = 'usd';

  test('CentOS or Ubuntu linux (instance without savings plan)', (done) => {
    const operatingSystem = 'linux';

    assert(done, culture, currency, operatingSystem, 'S96');
  });

  test('Ubuntu Advantage Essential (no hybrid benefit)', (done) => {
    const operatingSystem = 'ubuntu-advantage-essential';

    assert(done, culture, currency, operatingSystem);
  });
});
