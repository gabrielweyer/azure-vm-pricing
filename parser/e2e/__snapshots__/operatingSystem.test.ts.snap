// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`End-to-end tests for supported operating systems CentOS or Ubuntu linux (instance without savings plan) 1`] = `"S96,96,768,N/A,N/A,4.0808,N/A,2.0712,N/A,N/A,N/A,N/A,N/A,N/A,N/A"`;

exports[`End-to-end tests for supported operating systems Ubuntu Advantage Essential (no hybrid benefit) 1`] = `"D2 v3,2,8,0.126,N/A,0.0662,N/A,0.0598,N/A,0.0207,N/A,0.1103,N/A,0.089,N/A"`;
